main.tf

```diff
@@ -156,11 +156,16 @@ module "openshift_install" {
   ocp_virtualization_enable = var.ocp_virtualization_enable
 }
 
+
 resource "null_resource" "get_kubeconfig" {
   depends_on = [module.prepare_openshift.finished]
 
   provisioner "local-exec" {
-    command = "mkdir -p ${path.root}/auth; scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${module.sshkey.ssh_private_key_file} root@${module.bastion.lb_ip}:/tmp/artifacts/install/auth/* ${path.root}/auth/"
+    command = <<EOT
+      [[ -d ${path.root}/auth ]] || mkdir -p ${path.root}/auth
+      /usr/bin/scp -r -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${module.sshkey.ssh_private_key_file} root@${module.bastion.lb_ip}:/tmp/artifacts/install/auth/ ${path.root}/auth/
```

Comment:
`-r` was used instead of `*` because I wanted to ensure that shell expansion of the glob wasn't at fault.  I don't believe it was, but `-r` is safer for this purpose.